### PR TITLE
Metadata improvements

### PR DESCRIPTION
**Pull request by aleksanb**
Wow such improve much great.
Creating tooling is such fun!

### CODE REVIEW

File: nin/backend/blank-project/index.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    startButton.innerHTML = 'Starting..!';
+    setTimeout(function() {
+      wrapper.classList.remove('visible');
+      wrapper.style.background = '#fff';
+      document.body.style.background = '#000';
+      demo.start();
+    }, 100);
+  };
+  </script>
+  <!--<script>
```
Comment (sigvef) [inline nin/backend/blank-project/index.html]: Commented script?

Comment (aleksanb) [inline nin/backend/blank-project/index.html]: I was unsure whether you want google analytics bundled by default, and in that case what analytics ID you should use.
I can make it a default option, and then template in a GA key from nin.json.

Comment (sigvef) [inline nin/backend/blank-project/index.html]: That sounds good.

**Comment by aleksanb** [comment]
Ping. All operational. Tested against newest dix with both https://github.com/ninjadev/dix/pull/139 and https://github.com/ninjadev/dix/pull/138 merged, and both the .html and .html.png have correct metadata now and run in the browser fine.

**Pull request merged**